docs/source/content/faqs.md

```diff
@@ -0,0 +1,44 @@
+# Frequently Asked Questions [FAQs]
+
+Over the years, we have gathered a valuable experience running SHARPy so we would like to collect here a few of the
+most frequently asked questions we get here to hopefully help other users.
+
+In addition to the questions listed below, do check the [issues](https://github.com/ImperialCollegeLondon/sharpy/issues?q=is%3Aissue) page in our GitHub repo as you may find useful 
+information there. Do also check our [Short Debugging Guide](./debug.html).
+
+* __[Q] I get a `ModuleNotFound Error` when trying to run SHARPy.__
+
+    _[A]_ Make sure you have loaded the SHARPy variables using the command `source <path_to_sharpy>/bin_sharpy_var.sh`.
+    
+* __[Q] When plotting the aerodynamic forces in Paraview from the UVLM, the forces at the boundary between two 
+surfaces (for instance at the wing root) appears halved. Is my simulation incorrect?__
+
+    _[A]_ This is most likely not an issue with your simulation. We have observed over time that Paraview actually only
+    plots the result from one of the surfaces, hence why it appears half of what it should be. If you extract the forces
+    without using Paraview using the 
+    [WriteVariablesTime](https://ic-sharpy.readthedocs.io/en/master/includes/postprocs/WriteVariablesTime.html) postprocessor you will get the correct result.
```

Comment:
@ArturoMS13, I wrote this a while back, do you know if this is correct? Or do the surfaces actually have half the force at a shared boundary?

Reply:
I have always though this was a visuallisation problem. When two surfaces match perfectly at two points, UVLM gives you half of the force on each surface. If you plot those forces on paraview you get two arrows one on top of each other. Visually you see a force which has half of the expected module, however you have two forces. As you say, when those forces are mapped onto the structure you get both contributions and, thus, the problem disappears.

My aerodynamic surfaces do not usually match but if you have an example I can doublecheck.

Reply:
Ok, so the actual force at the boundary is half and the issue is that Paraview doesn't add them up but just shows one.

Then I think my little snippet of text in the FAQs is correct